protocol/errors.json

```diff
@@ -182,6 +182,18 @@
   "91004": "unable to automatically re-enter presence channel",
   "91005": "presence state is out of sync",
   "91100": "member implicitly left presence channel (connection closed)",
+  "92000": "Invalid message type: state type required",
+  "92001": "Channel state max objects limit exceeded",
+  "92002": "State messages required",
+  "92003": "State message required",
```

Comment:
Can we unify this into a single code, `State message(s) required`?

Reply:
Yeah make sense